package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
+    "@react-native-community/eslint-config": "^1.0.0",
```

Comment:
What rules is this adding? and should it be added to ``extends`` in ``.eslintrc.yml``?

Also is there some way to generate all the rules from our ``.eslintrc.yml`` 
so we can see how the packages we are extending eg.
 - airbnb
 - prettier
 - prettier/react

are affecting the resulting rules?

Reply:
The reason I added this was because it was initially in an exdtension in the eslintrc. It appears this has changed substantially since I last looked and this has therefore become irrelevant :)